src/components/technologies/genericTechnologies/conv1.py

```diff
@@ -838,25 +839,73 @@ def bind_disjunctions(dis, t):
             )
 
         else:
+            if data["config"]["optimization"]["typicaldays"]["N"]["value"] == 0:
+                input_aux = self.input
```

Comment:
could we refactor this var into input_aux_rr or something similiar to not confuse it with input_aux in the technology class?

Reply:
done